task/tkn-bundle-oci-ta/0.1/tkn-bundle-oci-ta.yaml

```diff
@@ -87,11 +93,11 @@ spec:
             neg=1
             path="${path#\!}"
           fi
-          if [[ -d "/var/workdir/source/${path}" ]]; then
+          if [[ -d "/var/workdir/source/${SOURCE_CODE_DIR}/${path}" ]]; then
```

Comment:
I don't think this is correct for `oci-ta`. We download the source artifact like this

```
$(params.SOURCE_ARTIFACT)=/var/workdir/source
```

And the source artifact is just the repo, it doesn't have another `./source` subdirectory.

Most TA recipes do the replacement like this

https://github.com/konflux-ci/build-definitions/blob/4692a3811cf2cf319208729aa864b186f5f3743d/task/prefetch-dependencies-oci-ta/0.2/recipe.yaml#L51-L52

Reply:
Or like this, this may be better

https://github.com/konflux-ci/build-definitions/blob/4692a3811cf2cf319208729aa864b186f5f3743d/task/buildah-oci-ta/0.4/recipe.yaml#L13-L14

Reply:
Yes, I reverted the oci-ta to what it was and apply the changes only for `tkn-bundle`

Reply:
I now see that the CI is failing because its running the `generate-ta-tasks.sh` and builds the `tkn-bundle-oci-ta` task from the `tkn-bundle` task

Reply:
@chmeliik Can you PTAL.
I updated the recipe as in buildah task and updated the oci-ta task.